src/test/groovy/steps/DataAnalyticsSteps.groovy

```diff
@@ -37,7 +31,7 @@ class DataAnalyticsSteps {
 
     def http200(count, calls){
         while (count <= calls) {
-            Response http200 = repoSteps.getRepos(username, password)
+            Response http200 = repoSteps.getRepos(artifactoryURL, username, password)
```

Comment:
Repeatedly putting the same "variable" that's actually "constant" into a function is a pattern for errors. Groovy is really great at eliminating this without creating a class:

```groovy
def getRepos = (repoSteps.&getRepos).curry(artifactoryURL,username,password)
```

Reply:
Done. curry does the magic